### PR TITLE
Don't log secret of s3.

### PR DESCRIPTION
**Pull request by YuJuncen**
### What problem does this PR solve? 
fix [bug hunting #71](https://github.com/tidb-challenge-program/bug-hunting-issue/issues/71)

### What is changed and how it works?

I change the line that logging the secret information:

https://github.com/pingcap/br/blob/00fe08c83ed64f8e0461577f1310dc9eed8f0407/pkg/backup/client.go#L763

### Check List 

Tests 

 - Unit test
 - Integration test
 - No code

Code changes

 - Has exported function/method change(move `LogArguments` from `utils` to `task`).

### Release Note

 - No release note

### CODE REVIEW

File: pkg/backup/client.go [hunk 1]
```diff
@@ -760,7 +760,12 @@ func SendBackup(
 	req kvproto.BackupRequest,
 	respFn func(*kvproto.BackupResponse) error,
 ) error {
-	log.Info("try backup", zap.Any("backup request", req))
+	log.Info("try backup",
+		zap.Binary("StartKey", req.StartKey),
+		zap.Binary("EndKey", req.EndKey),
+		zap.Uint64("RateLimit", req.RateLimit),
+		zap.Uint32("Concurrency", req.Concurrency),
```
Comment (kennytm) [inline pkg/backup/client.go:767]:
```suggestion
		zap.Uint64("storeID", storeID),
```

These two are fixed and can be derived from the CLI args, so they're not worth logging (but store ID is likely more useful for debugging).

File: pkg/task/common.go [hunk 2]
```diff
@@ -286,3 +290,27 @@ func escapeFilterName(name string) string {
...
+		return zap.Stringer(f.Name, hiddenQuery)
+	}
+	return zap.Stringer(f.Name, f.Value)
+}
+
+// LogArguments prints origin command arguments.
+func LogArguments(cmd *cobra.Command) {
+	var fields []zap.Field
+	cmd.Flags().VisitAll(func(f *pflag.Flag) {
+		fields = append(fields, flagToZapField(f))
```
Comment (kennytm) [inline pkg/task/common.go:313]: oh, could we skip the log if the flag isn't explicitly provided?

```suggestion
		if f.Changed {
			fields = append(fields, flagToZapField(f))
		}
```

File: pkg/task/common_test.go [hunk 3]
```diff
@@ -0,0 +1,37 @@
+package task
```
Comment (overvenus) [inline pkg/task/common_test.go:1]: Please add license header.

File: pkg/task/common.go [hunk 4]
```diff
@@ -6,9 +6,13 @@ import (
 	"context"
 	"crypto/tls"
 	"fmt"
+	"net/url"
 	"regexp"
 	"strings"
 
+	"github.com/pingcap/log"
+	"go.uber.org/zap"
+
```
Comment (overvenus) [inline pkg/task/common.go:15]: Please remove the empty line.

**Pull request merged**